src/main/java/com/commercetools/pingpong/model/Player.java

```diff
@@ -64,4 +79,7 @@ public int getMatchScore() {
         return matchScore;
     }
 
+    public boolean amIServing() {return serve;}
+
+    public boolean didIServeAtBeginning() {return beginningServe;}
```

Comment:
Why do we need to differentiate between beginning serve and serve? Is that really necessary?

Reply:
Yeah, I think it is ... maybe you know why after seeing the fix:
a8405c05a7c0eccf1f19cb29b4f66795c2c83985

Reply:
True 👍 Thanks for clarifying.